sdv/metadata/table.py

```diff
@@ -769,6 +769,8 @@ def from_dict(cls, metadata_dict, dtype_transformers=None):
             entity_columns=metadata_dict.get('entity_columns') or [],
             context_columns=metadata_dict.get('context_columns') or [],
             dtype_transformers=dtype_transformers,
+            min_value=metadata_dict.get('min_value') or 'auto',
+            max_value=metadata_dict.get('max_value') or 'auto',
```

Comment:
Maybe we should also add the `rounding` argument?

Reply:
I did try adding the rounding argument at first, but it seems to crash because of the nans issue (https://github.com/sdv-dev/RDT/pull/244).

We could wait for that PR to be merged first